dwave/cloud/config.py

```diff
@@ -48,171 +48,141 @@
-    >>> client.default_solver   # doctest: +SKIP
-    'EXAMPLE_2000Q_SYSTEM_A'
-    >>> client.endpoint  # doctest: +SKIP
-    'https://int.se.dwavesystems.com/sapi'
-    >>> client.token  # doctest: +SKIP
-    'DEF-987654321987654321987654321'
+    >>> print(client.endpoint)      # doctest: +SKIP
+    https://eu-central-1.cloud.dwavesys.com/sapi/v2/
+    >>> print(client.token)  # doctest: +SKIP
+    DEF-987654321987654321987654321
     >>> # code that uses client
     >>> client.close() # doctest: +SKIP
 
-    The next example uses :func:`~dwave.cloud.config.load_config` to load profile values.
-    **Most users do not need to use this method.** It loads from the following configuration
-    file, dwave_c.conf, located in the current working directory, and specified explicitly::
+    *   **Advanced Example:** Loading profile values
+       
+    This example uses :func:`~dwave.cloud.config.load_config` to load profile 
+    values. **Most users do not need to use this method.** It loads from the 
+    following configuration file, ``dwave_c.conf``, located in the current 
+    working directory, and which contains two profiles in addition to the 
+    defaults section::
 
         [defaults]
-        endpoint = https://url.of.some.dwavesystem.com/sapi
+        token = ABC-123456789123456789123456789
         solver = {"qpu": true}
 
-        [dw2000a]
-        solver = {"software": true, "name": "EXAMPLE_2000Q"}
-        token = ABC-123456789123456789123456789
+        [advantage-a]
+        solver = {"software": true, "name": "Fake_Advantage"}
 
-        [dw2000b]
+        [advantage-b]
         solver = {"qpu": true}
         token = DEF-987654321987654321987654321
-
-    This configuration file contains two profiles in addition to the defaults section.
-    In the following example code, first no profile is specified, and the first profile
-    after the defaults section is loaded with the solver overridden by the environment
-    variable. Next, the second profile is selected
-    with the explicitly named solver overriding the environment variable setting.
+   
+    The following example code first loads, by default, the first profile 
+    (``[advantage-a]``) after the ``[defaults]`` section with its solver 
+    preference overridden by the environment variable. Next, the second profile
+    (``[advantage-b]``) is explicitly selected with the explicitly named solver
+    overriding the environment variable setting.
 
     >>> import dwave.cloud
     >>> import os
-    >>> os.environ['DWAVE_API_SOLVER'] = 'EXAMPLE_2000Q_SYSTEM'   # doctest: +SKIP
+    >>> os.environ["DWAVE_API_SOLVER"] = "Advantage_Emulator"   # doctest: +SKIP
     >>> dwave.cloud.config.load_config("./dwave_c.conf")   # doctest: +SKIP
-    {'client': 'sw',
-     'endpoint': 'https://url.of.some.dwavesystem.com/sapi',
-     'proxy': None,
-     'headers': None,
-     'solver': 'EXAMPLE_2000Q_SYSTEM',
+    {'solver': '{"software": true, "name": "Advantage_Emulator"}',
```

Comment:
Haven't checked, but this should be:
```suggestion
    {'solver': 'Advantage_Emulator'},
```
If it's not, that might be a bug. Solver from the env should replace the complete solver definition, not just patch partially (name).

Reply:
Dropped example